tests/rados/test_scrub_log.py

```diff
@@ -17,94 +18,238 @@
     pool_configs = config["pool_configs"]
     pool_configs_path = config["pool_configs_path"]
+    verify_log_check = config["verify_log"]
     installer = ceph_cluster.get_nodes(role="installer")[0]
-
+    debug_increased_file_size = ""
+    warn_increased_file_size = ""
     regex = r"\s*(\d.\d)-rhel-\d"
     build = (re.search(regex, config.get("build", config.get("rhbuild")))).groups()[0]
     if not float(build) >= 5.3:
         log.info(
             "Test running on version less than 5.3, feature not present, skipping test"
         )
         return 0
+    parm_out_put_check = None
+    # Check that paramters are removed from the build
+    if verify_log_check:
+        try:
+            parm_out_put_check = mon_obj.get_config(
+                section="mon", param="mon_cluster_log_file_level"
+            )
+        except Exception as e:
+            log.error(f"An unexpected error occurred: {e}")
+        if parm_out_put_check is not None:
+            log.error(
+                f"The mon_cluster_log_file_level parameter not deleted and the value is {parm_out_put_check}"
+            )
+            return 1
+        try:
+            parm_out_put_check = mon_obj.get_config(
+                section="mon", param="mon_cluster_log_to_syslog_level"
+            )
+        except Exception as e:
+            log.error(f"An unexpected error occurred: {e}")
+        if parm_out_put_check is not None:
+            log.error(
+                f"The mon_cluster_log_to_syslog_level parameter not deleted and the value is {parm_out_put_check}"
+            )
+            return 1
+        try:
+            parm_out_put_check = mon_obj.get_config(
+                section="mon", param="mon_cluster_log_level"
+            )
+        except Exception as e:
+            log.error(f"An unexpected error occurred: {e}")
 
+        parm_out_put_check = parm_out_put_check.strip()
+        if parm_out_put_check != "debug":
+            log.error(
+                f"The mon_cluster_log_level default expected value should be debug."
+                f"The actual value is {parm_out_put_check}"
+            )
+            return 1
     try:
-        with open(pool_configs_path, "r") as fd:
-            pool_conf_file = yaml.safe_load(fd)
-
-        pools = []
-        acting_sets = {}
-        for i in pool_configs:
-            pool = pool_conf_file[i["type"]][i["conf"]]
-            create_given_pool(rados_obj, pool)
-            rados_obj.bench_write(
-                pool_name=pool["pool_name"],
-                max_objs=50,
-                check_ec=False,
-                rados_write_duration=20,
+        for log_level in ["debug", "warn"]:
+            log.info(
+                f"====START:Testing the scenario with the mon_cluster_log_level value {log_level} ======="
+            )
+            # Get the ceph log size before starting the test
+            before_test_log_size = get_file_size(rados_obj, installer)
```

Comment:
I believe that the logs like "ceph.log" whose size is checked here, is only generated when "mon_cluster_log_to_file" param is true. Can you confirm if that is indeed the case? 

If Yes, Please make sure to enable the param on the cluster before starting the tests.

Note : As part of def log_to_file() , we are enabling the param, but incase that method is not called during cluster set-up, the logs might not be present on the cluster. better to make sure.

Reply:
Included the enable_file_logging method in the code.